projects/observability/src/pages/explorer/explorer-dashboard-builder.ts

```diff
@@ -570,34 +570,32 @@ export class ExplorerDashboardBuilder {
     }
   }
 
-  private removeColumnsDuplicated(columns: ModelJson[]): ModelJson[] {
-    return uniqBy(columns, column => (column?.value as AttributeValue).attribute ?? '');
+  private removeColumnsDuplicated(columns: ModelSpecificationJson[]): ModelJson[] {
+    return uniqBy(columns, column =>
+      column.value?.type === ValueSpecificationType.Attribute ? column.value.attribute : column
+    );
   }
 
   private getGeneratedTableColumns(
     attributes: AttributeMetadata[],
     context: ExplorerGeneratedDashboardContext,
     selectedProperties: Specification[]
-  ): ModelJson[] {
-    const attributesToExclude = this.getAttributesToExcludeFromUserDisplay(context);
```

Comment:
We need to be careful about removing this. Functionally uniq check should take its place - but please go through the attributes and make sure there's nothing in there that wouldn't be covered by uniq. I believe this is also protected so subclasses can override, so we need to consider that as well and check any subclasses.

Reply:
Calls and Status Code are present after changing this and they should not be there. So I will keep the code for now.